books/RayTracingTheRestOfYourLife.html

```diff
@@ -18,15 +18,27 @@
 
 In _Ray Tracing in One Weekend_ and _Ray Tracing: the Next Week_, you built a “real” ray tracer.
 
-In this volume, I assume you will be pursuing a career related to ray tracing, and we will dive into
-the math of creating a very serious ray tracer. When you are done you should be ready to start
-messing with the many serious commercial ray tracers underlying the movie and product design
-industries. There are many many things I do not cover in this short volume; I dive into only one of
-many ways to write a Monte Carlo rendering program. I don’t do shadow rays (instead I make rays more
-likely to go toward lights), bidirectional methods, Metropolis methods, or photon mapping. What I do
-is speak in the language of the field that studies those methods. I think of this book as a deep
-exposure that can be your first of many, and it will equip you with some of the concepts, math, and
-terms you will need to study the others.
+If you are motivated, you can take the source and information contained in those books to implement
+any visual effect you want. The source provides a meaningful and robust foundation upon which to
+build out a raytracer for a small hobby project. Most of the visual effects found in commercial ray
+tracers rely on the techniques described in these first two books. However, your capacity to add
+increasingly complicated visual stuffs like subsurface scattering or nested dielectrics will be
+severely limited by a missing mathematical foundation. In this volume, I assume that you are either
+a highly interested student, or are someone who is pursuing a career related to ray tracing. We will
+be diving into the math of creating a very serious ray tracer. When you are done you should be ready
+to start messing with the many serious commercial ray tracers underlying the movie and product
+design industries.
+
+There are many many things I do not cover in this short volume. There are many ways of writing Monte
+Carlo rendering programs, I dive into only one of them. I don’t cover shadow rays (instead deciding
```

Comment:
“... rendering programs, I dive ...”. Replace comma with semi-colon, period, or em-dash.

Reply:
I love the em-dash